gapis/replay/export_replay.go

```diff
@@ -88,7 +88,7 @@ func (m *exportManager) Export(ctx context.Context, waitRequests int) (*gapir.Pa
 	}
 	ctx = log.V{"replay target ABI": replayABI}.Bind(ctx)
 
-	b := builder.New(replayABI.MemoryLayout)
+	b := builder.New(replayABI.MemoryLayout, nil)
```

Comment:
This probably won't work, since we no longer have the initial commands in Replay(.

Fix: Either generate / store a new capture that contains the initial commands, or detect this in the replay(.

Reply:
This should be fixed by latest.